Update index.md

jayunit100 opened the pull request:
## Description

The landing has a couple of minor typos, this tightens it up a little bit. Documentation typo fix + a few other readability fixes.

First PR into calico, more to come as I get a little more familiar with things :).

## Related issues/PRs

```release-note
Front page doc improvements.
```

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/projectcalico/calico?pullRequest=2875) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/projectcalico/calico?pullRequest=2875) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/projectcalico/calico?pullRequest=2875) it.</sub>

caseydavenport commented:
Deploy preview for *calico* ready!

Built with commit 6e96d315a93c14c82382b74081bc34dfa720b4ee

https://deploy-preview-2875--calico.netlify.com

caseydavenport commented:
Deploy preview for *calico* ready!

Built with commit 6e96d315a93c14c82382b74081bc34dfa720b4ee

https://deploy-preview-2875--calico.netlify.com

caseydavenport commented on v3.9/introduction/index.md:
```diff
@@ -43,7 +43,7 @@ custom_css: css/intro.css
   <div class="col-md-6">
     <h3 style="margin-top: 5px">Performance</h3>
-    <p>{{site.prodname}} uses the Linux kernel’s built-in highly optimized forwarding and access control capabilities to deliver native Linux networking dataplane performance, typically without requiring any of the encap/decap overheads associated with first generation SDN networks. {{site.prodname}}’s control plane and policy engine has been fine tuned over many years of production use to minimi ...
+    <p>{{site.prodname}} uses the Linux kernel’s built-in, highly optimized forwarding and access control capabilities to deliver native Linux networking dataplane performance (note that this is done in {{site.prodname}} *without* requiring any of the encap/decap overheads associated with first generation SDN networks). {{site.prodname}}’s control plane and policy engine has been fine-tuned over  ...
```
I think we should revert this diff as well, as I don't think it fixes any real problem in the existing text.

jayunit100 replied:
ok, specifically I was referring to the performance tests which were done relative to other CNIs, https://itnext.io/benchmark-results-of-kubernetes-network-plugins-cni-over-10gbit-s-network-36475925a560 .  IMO indicating that the 'fine-tuning' has been verified with actual measurements has some value...  but we can worry about that in a later PR maybe.

jayunit100 commented on v3.9/introduction/index.md:
```diff
@@ -28,7 +28,7 @@ custom_css: css/intro.css
     <h3 style="margin-top: 5px">Best practices for network security</h3>
     <p>{{site.prodname}}’s rich network policy model makes it easy to lock down communication so the only traffic that flows is the traffic you want to flow. You can think of {{site.prodname}}’s security enforcement as wrapping each of your workloads with its own personal firewall that is dynamically re-configured in real time as you deploy new services or scale your application up or down.</p>
-    <p>{{site.prodname}}’s policy engine can enforce the same policy model at the host networking layer and (if using Istio & Envoy) at the service mesh layer, protecting your infrastructure from compromised workloads and protecting your workloads from compromised infrastructure.</p>
+    <p>{{site.prodname}}’s policy engine can enforce the same policy model at the host networking layer and (if using Istio & Envoy) at the service mesh layer, thus protecting infrastructure from potentially comprimised workloads (while also protecting workloads from any rogue elements in your infrastructure).</p>
```
sure, will do.  "comprimised infrastructure" seems vague to me, but i suppose its not too important.

caseydavenport replied:
Yep. It is vague in that it covers a number of different things - adding the "potentially" doesn't make it any less vague, though.

jayunit100 commented:
I simplified the PR, and rather then clarifying the CPU part, converted it to a hyperlink pointing to `<a href="https://itnext.io/benchmark-results-of-kubernetes-network-plugins-cni-over-10gbit-s-network-36475925a560">minimize overall CPU usage and occupancy</a>`  ..  Thanks for the feedback casey .  Its been a while .. just rejoining upstream K8s now...  so looking forward to helping more.  Please do point me at help wanted issues that I can jump in on.

caseydavenport commented on v3.9/introduction/index.md:
```diff
@@ -43,7 +43,7 @@ custom_css: css/intro.css
   <div class="col-md-6">
     <h3 style="margin-top: 5px">Performance</h3>
-    <p>{{site.prodname}} uses the Linux kernel’s built-in highly optimized forwarding and access control capabilities to deliver native Linux networking dataplane performance, typically without requiring any of the encap/decap overheads associated with first generation SDN networks. {{site.prodname}}’s control plane and policy engine has been fine tuned over many years of production use to minimi ...
+    <p>{{site.prodname}} uses the Linux kernel’s built-in highly optimized forwarding and access control capabilities to deliver native Linux networking dataplane performance, typically without requiring any of the encap/decap overheads associated with first generation SDN networks. {{site.prodname}}’s control plane and policy engine has been fine tuned over many years of production use to <a hre ...
```
I don't think we should link to this report.

Firstly, it's out of date and there is an updated report. But also, while it does seem a fair comparison, performance testing is hard, there are a lot of similar reports on performance, and I'd rather not "endorse" one report or another in this way.

jayunit100 replied:
Ok fair enough. I’ll remove that part :).

jayunit100 replied:
ok,... i deleted the `a href` and alluded to what you said above.  that way the statement isnt entirely anecdotal.. :) let me know if that works better (if not ill just delete this sentence change i think)

caseydavenport commented:
> Its been a while .. just rejoining upstream K8s now... so looking forward to helping more. Please do point me at help wanted issues that I can jump in on.

Welcome back! Feel free to reach out on Slack in the #calico-dev channel and we can chat about areas that could use your help :)

jayunit100 commented:
ok, updated the patch to remove the specific `a href` and alluded that there were some third-party studies referencing the performance that are "interesting" to read :).

jayunit100 commented:
Hi folks, 
- i also added ... 
```
 (there are interesting articles about this online conducted by various external parties which attempt to objectively illustrate this metric under various scenarios, but we do not endorse any one benchmark specifically in our official documentation)
```
to clarify why we don't specifically call out any performance benchmarks, 
- and created #2912 , which might be an eventual follow on.

caseydavenport commented:
>  (there are interesting articles about this online conducted by various external parties which attempt to objectively illustrate this metric under various scenarios, but we do not endorse any one benchmark specifically in our official documentation)

I'm sorry to be such a pain about this, but I don't think this is something we want to say in this doc. Mainly, I'm not convinced it adds value to any new users who are reading this and it feels a bit out of place in this introductory / marketing content. 

For some additional context into my thinking, this introduction page is meant to showcase what's great about Calico in a short and concise way - the fewer words, the better. The journey for a new user coming to this page isn't "Let's see if I can find a detailed performance comparison". Rather, it's "What is Calico?"

jayunit100 commented:
no problem casey, ill update it now :)

jayunit100 commented:
@caseydavenport ok, minimized the fixes.  I'll bikeshed about the performance stuff with you in another thread later.  Thanks again for your patience.

caseydavenport commented:
LGTM - thanks for bearing with me :)
